css/css-text-decor/reference/text-decoration-propagation-05-ref.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<title>CSS Test: text-decoration propagation - Tables</title>
+<link rel="author" title="Timothy Loh" href="mailto:timloh@chromium.org" />
+<link rel="help" href="https://www.w3.org/TR/css-text-decor-3/#line-decoration">
+<link rel="match" href="reference/text-decoration-propagation-05-ref.html">
```

Comment:
No need to point to the ref in the reference file.

Reply:
Just fixed it in next push.